web/components/charts/contract/choice.tsx

```diff
@@ -22,7 +22,7 @@ import { Row } from 'web/components/layout/row'
 import { Avatar } from 'web/components/avatar'
 
 // thanks to https://observablehq.com/@jonhelfman/optimal-orders-for-choosing-categorical-colors
-const CATEGORY_COLORS = [
+export const CATEGORY_COLORS = [
```

Comment:
I still think these colors are super ugly :( Gavriel suggests some alternatives https://discord.com/channels/915138780216823849/915138780653051910/1013157893593370664, and I like the look of https://projects.susielu.com/viz-palette

Reply:
(but can change this in a different PR. it's mostly ugly on the chart, which isn't changing in this PR)

Reply:
haha I agree, Marshall and I are actually thinking of only displaying the top 5 answers as colors, the reset in a gray color under 'Other'
I found this palette that is specifically colorblind friendly
![image](https://user-images.githubusercontent.com/46611122/193917565-c3ec6178-5799-4cd4-be07-63a26b4adf0b.png)